rr_viz/res/cfg/nav_view.rviz

```diff
@@ -77,14 +72,98 @@ Visualization Manager:
         Expand Link Details: false
         Expand Tree: false
         Link Tree Style: Links in Alphabetic Order
+        base_footprint:
```

Comment:
All this is .rviz visualisation config (like how you save when file save) Because the extrm description was openend when saving the template, thats why it has them. I think it will change automatically on run to the current robots description